src/views/DraftDetail.vue

```diff
@@ -342,24 +381,63 @@ async function findProduct() {
   isSearchingProduct.value = false
 }
 
-async function addProductToCount() {
-  // If product is not found in the searched string then do not make the api call
-  // This check is only required to handle the case when user presses the enter key on the input and we do not have any result in the searchedProduct
-  if(!searchedProduct.value.productId || !queryString.value) {
-    return;
+async function findProductFromIdentifier(payload: any) {
+  
+  const idType = payload.idType;
+  const idValues = payload.idValue;
+  const filterString = `${idType}: (${idValues.join(' OR ')})`;
+
+  try {
+    const resp = await ProductService.fetchProducts({
```

Comment:
Instead of making service call here, we should think of using the existing action for fetching products. Need to go with this approach as after searching for some products we store them in state so that the same products are not searched again, but in this case the same products will be searched twice once from here and then once after adding them to count from the search product action.